src/other/tables.jl

```diff
@@ -2,9 +2,10 @@ Tables.istable(::Type{<:AbstractDataFrame}) = true
 Tables.columnaccess(::Type{<:AbstractDataFrame}) = true
 Tables.columns(df::AbstractDataFrame) = df
 Tables.rowaccess(::Type{<:AbstractDataFrame}) = true
-Tables.rows(df::AbstractDataFrame) = Tables.rows(columntable(df))
+Tables.rows(df::AbstractDataFrame) = eachrow(df)
 
 Tables.schema(df::AbstractDataFrame) = Tables.Schema(names(df), eltype.(eachcol(df)))
+Tables.schema(df::DataFrameRows) = Tables.schema(getfield(df, :df))
```

Comment:
Maybe this needs a new test?

Reply:
Agreed - this is probably clear what it will produce, but a test will provide us with an error if in the future someone makes a breaking change in this part of source code.

Reply:
I went ahead and merged @tkf's PRs that added tests for `Tables.schema(df::DataFrameRows)` already, so we should be good to do here now.

Reply:
You mean https://github.com/JuliaData/DataFrames.jl/pull/2054? I don't see how it tests `schema`.

Reply:
No, https://github.com/JuliaData/DataFrames.jl/pull/2055

Reply:
Ah, OK. TBH I'd have preferred if you had waited a bit more until we had a chance to comment on the final design...

Reply:
Oops. I didn't realize there was more to discuss or pending questions/concerns. Is there something specific you're still wondering about?

Reply:
Actually, I have: https://github.com/JuliaData/DataFrames.jl/pull/2055#issuecomment-566421559. Sorry, it turned out I didn't understand what I wanted...